framework/framework_cancel_test.go

```diff
@@ -113,7 +87,7 @@ func Test_ProcessDelete(t *testing.T) {
 	}
 
 	for i, tc := range testCases {
-		err := ProcessDelete(tc.Ctx, nil, tc.Resources)
+		err := ProcessDelete(context.Background(), nil, tc.Resources)
```

Comment:
I tend to use `context.Background()` instead `context.TODO()` in tests, because it isn't really TODO and this messes with greping for TODOs.